test/src/blockMonitor/reorg.test.ts

```diff
@@ -123,9 +123,9 @@ class TestCase {
         );
 
     async traverse(reorgDetector: ReorgDetector, provider: asyncEmitTestProvider) {
-        const findReorgSpec = blockNumber =>
+        const findReorgSpec = (blockNumber: number) =>
             this.reorgs.filter(r => r.expectedAtBlockNumber === blockNumber && !r.observed)[0];
-        let currentReorg;
+        let currentReorg: any;
```

Comment:
I think this should be IReorgInfo

Reply:
Changed to `IReorgInfo | undefined`, since it might be used before initializing it.